browsers/nodejs.json

```diff
@@ -255,13 +255,27 @@
           "engine": "V8",
           "engine_version": "8.1"
         },
+        "14.3.0": {
+          "release_date": "2020-05-19",
+          "release_notes": "https://nodejs.org/en/blog/release/v14.3.0/",
+          "status": "retired",
+          "engine": "V8",
+          "engine_version": "8.3"
+        },
         "14.6.0": {
           "release_date": "2020-07-21",
           "release_notes": "https://nodejs.org/en/blog/release/v14.6.0/",
           "status": "retired",
           "engine": "V8",
           "engine_version": "8.4"
         },
+        "14.8.0": {
+          "release_date": "2020-08-11",
+          "release_notes": "https://nodejs.org/en/blog/release/v14.8.0/",
+          "status": "retired",
```

Comment:
```suggestion
          "status": "esr",
```

Reply:
14.8 is not most recent, 14.15 is. But then we probably don't want to add every version that we'll never use? What do you think?

Reply:
Yeah, I've generally stuck with the lowest versions for each column on https://node.green/ with some exceptions (such as major versions or support differences undocumented on node.green).